fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetaClient.java

```diff
@@ -119,8 +121,13 @@ public class AutoCloseClient implements AutoCloseable {
         private final IMetaStoreClient hiveClient;
 
         private AutoCloseClient(HiveConf conf) throws MetaException {
-            hiveClient = RetryingMetaStoreClient.getProxy(conf, dummyHookLoader,
-                    HiveMetaStoreThriftClient.class.getName());
+            if ("dlf".equalsIgnoreCase(conf.get("hive.metastore"))) {
```

Comment:
Where is the 'hive.metastore' configuration defined ? Could the name be modified? I think the 'hive.metastore' means too much